pacta/pacta.go

```diff
@@ -260,6 +260,27 @@ func (o *Blob) Clone() *Blob {
 	}
 }
 
+type BlobOwnerInformation struct {
```

Comment:
niiiit: `Information` is a long word that ironically adds little information, just `BlobOwner` is probably fine. Even if it means your `bois` variables become a less-entertaining `bos`